Misc/NEWS.d/next/C API/2020-01-17-19-25-48.bpo-39372.hGJMY6.rst

```diff
@@ -0,0 +1,9 @@
+Clean header files of interfaces defined but with no implementation. The
+public API symbols being removed are:
+
```

Comment:
I afraid that using multi-paragraph news entry can cause problems when files be merged into a single NEWS file.

Reply:
Thanks for pointing that out.... it was too good to be true (I couldn't check at the time I pushed this change).